tests/appsec/iast/test_iast_no_request.py

```diff
@@ -0,0 +1,51 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import BaseTestCase, missing_feature, interfaces, coverage, released
+from utils.interfaces._library._utils import get_root_spans
+from utils.interfaces._core import BaseValidation
+
+
+@coverage.basic
+@released(
+    dotnet="?", golang="?", java="?", nodejs="?", php_appsec="?", python="?", ruby="?", cpp="?",
```

Comment:
Java should already be supported, so you can add the release version here?

Reply:
Done, sorry for a long long delay